src/he/utils.h

```diff
@@ -0,0 +1,49 @@
+/* *
+ * Lightway Core
+ * Copyright (C) 2022 Express VPN International Ltd.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+/**
+ * @file utils.h
+ * @brief Utility functions for ease of use
+ *
+ */
+
+#ifndef UTILS_H
+#define UTILS_H
+
+#include <he.h>
```

Comment:
Use `#include "he.h"`, not `<he.h>` because we want to only include the "include/he.h" from the local project directory, not from the standard include search paths.

Reply:
Every other file in the project includes it as `<he.h>` do we want to have it all changed?

Reply:
OK, let's fix all them together in a separate PR then. I see only "memory.h" is doing it right.